Packs/PAN-OS/ReleaseNotes/2_2_2.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Palo Alto Networks PAN-OS
+
+- Fixed an issue in which the `pan-os-edit-rule` command would fail for some elements.
```

Comment:
```suggestion
- Fixed an issue in which the ***pan-os-edit-rule*** command would fail for some elements.
```
[release notes examples](https://xsoar.pan.dev/docs/documentation/release-notes#examples)